ref_slam_node:
  ros__parameters:
    map_save:
      save_path: "."
      image_width: 2000
      image_height: 2000
      origin_x: -50.0 #m
      origin_y: -50.0 #m
      resolution: 0.05 #m/pixel
    ref_extractor:
      radius: 0.035
      max_radius: 0.04
      min_radius: 0.03
      cluster_threshold: 0.1
      match_threshold: 1.0
      max_iterations: 10
      identify_threshold: 0.8

    map_optimizer:
      window_size: 20
      duplicates_threshold: 1.0
      keyframe_distance: 0.5
      keyframe_angle: 3.0 # degrees
      ref_share_count: 4
    run_mode: loc # slam loc
    map_path: "/home/luqk/ros2/sim/zc_sim/zc_cr/map.yaml"
